### PR TITLE
remove sentiment-socrer and its integration-test

### PR DESCRIPTION
**Pull request by SaketJNU**
An action item for https://github.com/openshiftio/openshift.io/issues/645

### CODE REVIEW

**Comment by tisnik** [comment]
Hi @SaketJNU . I'd suggest to let the code in common.py untouched (it might be reused in the future, who knows ;) and remove the following lines from the features:

```
cd integration-tests/features
grep "I should find the proper sentiment values in the stack analysis response" *.feature
```

**Comment by GeorgeActon** [comment]
@SaketJNU @samuzzal-choudhury  Is this completed?

**Comment by fridex** [comment]
> @SaketJNU @samuzzal-choudhury Is this completed?

@SaketJNU please rebase on top of the current master to see if CI succeeds, there's a merge conflict.

**Comment by SaketJNU** [comment]
@GeorgeActon, @fridex  In sprint planning, it was decided as a less priority work, and it is not clear that in future sentiment-analysis will be a part of the stack-analysis. Still, it is left to complete. My proposal is that "we are not sure about sentiment-scorer then keep it in flow with one time execution mode, i.e. first time when the pkg. has no sentiment-score sentiment work-flow will run but next time in future, it will remain silent so it will not go to stackoverflow data to update values."  
If proposal look good then will close this and open only one PR with 1 line change in server repo.

**Comment by SaketJNU** [comment]
Conclusion for this PR is: to remove "sentiment-checker" entirely. An attempted is made in term of updations. Thanks.

**Comment by pkajaba** [comment]
[test]

**Comment by jpopelka** [comment]
[test this]

**Comment by jpopelka** [comment]
You need to remove the
`I should find the proper sentiment values in the stack analysis response`
also from `integration-tests/features/selfcheck.feature` [here](https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/features/selfcheck.feature#L13) and [here](https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/features/selfcheck.feature#L26)

File: integration-tests/features/steps/common.py [hunk 1]
```diff
@@ -1229,75 +1229,6 @@ def stack_analysis_check_licenses(context, licenses, path):
...
-                                                "result/0/user_stack_info")
-
-    alternate_node = recommendations_node["alternate"]
-    companion_node = recommendations_node["companion"]
-    dependencies_node = user_stack_info_node["dependencies"]
-
-    check_sentiment(alternate_node)
-    check_sentiment(companion_node)
-    check_sentiment(dependencies_node)
-
```
Comment (jpopelka) [inline integration-tests/features/steps/common.py]: [Python linter fails](https://cucos-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/fabric8-analytics-common-PRs/430/console) with
`integration-tests/features/steps/common.py:1233:1: E303 too many blank lines (3)`

Comment (SaketJNU) [reply integration-tests/features/steps/common.py]: removed one additional blank line.

**Comment by jpopelka** [comment]
`integration-tests/features/steps/common.py:1325:1: E302 expected 2 blank lines, found 1`

**Comment by SaketJNU** [comment]
I noticed the error shown in ```integration-tests/features/steps/common.py:1325:1: E302 expected 2 blank lines, found 1``` was fixed in earlier commit. So fixed one possible error in ```add a new line at line 1309``` commit.

**Pull request merged**